### PR TITLE
Add curriculum marketing text field

### PR DESCRIPTION
**Pull request by schenedx**
See the fields and API format below:

curriculumimage

curriculum

@sandroroux @iloveagent57 Please review

### CODE REVIEW

File: course_discovery/apps/api/tests/test_serializers.py [hunk 1]
```diff
@@ -670,13 +670,17 @@ def test_data(self):
         self.assertDictEqual(serializer.data, expected)
 
     def test_degree_marketing_data(self):
+        self.maxDiff = None
```
Comment (iloveagent57) [inline course_discovery/apps/api/tests/test_serializers.py]: delete this line.

**Review by iloveagent57 — COMMENTED** [review]
This all seems fine, you've just got a stray `maxDiff = None` in there still.

**Review by sandroroux — APPROVED** [review]
Great work.

File: course_discovery/apps/api/tests/test_serializers.py (hunk 1 above)
Comment (schenedx) [reply course_discovery/apps/api/tests/test_serializers.py]: @iloveagent57 Thank you thank you!

**Pull request merged**